packages/experiment-browser/src/config.ts

```diff
@@ -166,6 +173,7 @@ export const Defaults: ExperimentConfig = {
   instanceName: '$default_instance',
   fallbackVariant: {},
   initialVariants: {},
+  initialFlags: null,
```

Comment:
we should use `undefined` instead of null everywhere that is not already using null